docs/groovy/reference/table-operations/select/view.md

```diff
@@ -18,6 +18,7 @@ When using `view`, the data being requested is not stored in memory. Rather, a f
 
 > [!CAUTION]
 > When using `view` or [`updateView`](./update-view.md), non-deterministic methods (e.g., random numbers, current time, or mutable structures) produce _unstable_ results. Downstream operations on these results produce _undefined_ behavior. Non-deterministic methods should use [`select`](./select.md) or [`update`](./update.md) instead.
+> Concurrency control (serial marking and barriers) cannot be applied to `view` or [`updateView`](./update-view.md). These operations compute results on demand and cannot enforce ordering constraints. If you need serial evaluation or barriers, use [`select`](./select.md) or [`update`](./update.md) instead. See [Parallelizing queries](../../../conceptual/query-engine/parallelization.md#serialization) for more information.
```

Comment:
Should the link also appear at the bottom of the document?  I could go either way.  Should there also be a link to the parallelization quick start?

Reply:
It seems like these two should be different callouts because they cover different topics.